### PR TITLE
#7 Feat/init structure

### PR DESCRIPTION
**Pull request by lanhuong4867**
Create structure folders

https://github.com/GraphicDThanh/huong-le-nguyen-lan-internship/issues/7

### CODE REVIEW

File: practice-one/styles/components/all.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import url(btn.css);
+@import url(logo.css);
+@import url(nav-list.css);
```
Comment (GraphicDThanh) [inline practice-one/styles/components/all.css]: add one empty end line

File: practice-one/styles/base/typography.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@font-face {
+    font-family: "popins";
+    font-display: swap;
+    src: url("../../assets/fonts/Poppins-ExtraLight.ttf");
+}
+
```
Comment (GraphicDThanh) [inline practice-one/styles/base/typography.css:6]: Remove 1 redundant line

File: practice-one/styles/base/reset.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
```
Comment (GraphicDThanh) [inline practice-one/styles/base/reset.css:46]: code should use 2 spaces

File: practice-one/styles/base/reset.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
```
Comment (GraphicDThanh) [inline practice-one/styles/base/reset.css:25]: Update format, enter here.

File: practice-one/styles/abstracts/variables.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
+:root{
+    /* Color */
+    --normal-color: #ff5300;
+    --light-color: #fff7f4;
+    --white: #ffffff;
+    --secondary-color: #7b7b7b;
+    --dark-color: #373737;
+
+    /* Font-large */
+    --bigest: 49px;
```
Comment (GraphicDThanh) [inline practice-one/styles/abstracts/variables.css]: Typo, should be `biggest`. Add this extension to check typo English https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

File: practice-one/styles/components/btn.css [hunk 6]
```diff
@@ -0,0 +1,10 @@
+.btn{
```
Comment (GraphicDThanh) [inline practice-one/styles/components/btn.css]: check code format `.btn{` to `.btn {`

File: practice-one/styles/components/btn.css [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+    color: var(--dark-color);
+    font-size: var(--small);
+    font-weight: var(--bold-600);
+    background-color: none;
+    width: 130px;
+    height: 50px;
+    border-radius: 6%;
```
Comment (GraphicDThanh) [inline practice-one/styles/components/btn.css]: Use consistent unit, px here is better then %

File: practice-one/styles/pages/menu.css [hunk 8]
```diff
@@ -0,0 +1,5 @@
+.header__nav{
```
Comment (GraphicDThanh) [inline practice-one/styles/pages/menu.css]: We not use BEM naming convention, update to `header-nav`

**Comment by GraphicDThanh** [comment]
Please add README.md to run your app and editorconfig(read [here](https://editorconfig.org/)) 

Add code speller to check your code typo, [here](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)

File: practice-one/styles/components/all.css (hunk 1 above)
Comment (LanHuong142002) [reply practice-one/styles/components/all.css]: Thank you for your comment. I will fix it

File: practice-one/styles/base/typography.css (hunk 2 above)
Comment (LanHuong142002) [reply practice-one/styles/base/typography.css:6]: Thank you for your comment. I will fix it

File: practice-one/styles/base/reset.css (hunk 4 above)
Comment (LanHuong142002) [reply practice-one/styles/base/reset.css:25]: Thank you for your comment. I will fix it

File: practice-one/styles/abstracts/variables.css (hunk 5 above)
Comment (LanHuong142002) [reply practice-one/styles/abstracts/variables.css]: Thank you for your comment. I will fix it

File: practice-one/styles/base/reset.css (hunk 3 above)
Comment (LanHuong142002) [reply practice-one/styles/base/reset.css:46]: Thank you for your comment. I will fix it

File: practice-one/styles/components/btn.css (hunk 6 above)
Comment (LanHuong142002) [reply practice-one/styles/components/btn.css]: Thank you for your comment. I will fix it

File: practice-one/styles/components/btn.css (hunk 7 above)
Comment (LanHuong142002) [reply practice-one/styles/components/btn.css]: Thank you for your comment. I will fix it

File: practice-one/styles/pages/menu.css (hunk 8 above)
Comment (LanHuong142002) [reply practice-one/styles/pages/menu.css]: Thank you for your comment. I will fix it

**Comment by thanhnguyen-agilityio** [comment]
I closed this PR because it contains many tasks code as the same time. 
Please help me separate the code and create new PR for the task #7 only, thanks!

**Pull request closed without merging**